prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -52,7 +52,10 @@ presubmits:
       containers:
       - image: alpine:3.15.0
```

Comment:
The issue is within this image, the file not exists. IIUC, you should use gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master instead.

Reply:
You likely need to update this image if the test script is newly added to the repo.